pkgs/os-specific/linux/prl-tools/default.nix

```diff
@@ -25,14 +25,14 @@ assert (!libsOnly) -> kernel != null;
 assert lib.elem stdenv.hostPlatform.system [ "x86_64-linux" "i686-linux" "aarch64-linux" ];
```

Comment:
```suggestion
```
This should bet set in meta.platforms only and not prevent evaluation on other platforms.